Spec/PushActivationStateMachine.swift

```diff
@@ -240,19 +292,21 @@ class PushActivationStateMachine : QuickSpec {
                         guard let body = stateMachine.rest.defaultEncoder.decode(rawBody, error: nil) as? NSDictionary else {
                             fail("body is invalid"); return
                         }
+                        expect(request.httpMethod) == "POST"
                         expect(body.value(forKey: "id") as? String).to(equal(rest.device.id))
-                        expect(body.value(forKey: "push")).toNot(beNil())
-                        expect(body.value(forKey: "formFactor")).toNot(beNil())
-                        expect(body.value(forKey: "platform")).toNot(beNil())
+                        expect(body.value(forKey: "push") as? [String: [String: String]]).to(equal(["recipient": ["transportType": "apns"]]))
+                        expect(body.value(forKey: "formFactor") as? String) == "phone"
```

Comment:
Can you extract these expected values ("phone", ...) and store them in a `let` before running tests? That way tests are less stringly-typed :)

Reply:
Done 0935eae